src/main/java/com/sonyericsson/jenkins/plugins/bfa/ResultFilter.java

```diff
@@ -0,0 +1,20 @@
+package com.sonyericsson.jenkins.plugins.bfa;
+
+import hudson.model.Result;
+
+public class ResultFilter {
```

Comment:
`shouldScan` is already in `PluginImpl` so having `needToAnalyze` in there is better than a completely new class for just one method